common4j/src/main/com/microsoft/identity/common/java/util/IPlatformUtil.java

```diff
@@ -129,4 +130,12 @@ public interface IPlatformUtil {
      */
     @Nullable
     List<Map.Entry<String, String>> updateWithAndGetPlatformSpecificExtraQueryParameters(@Nullable List<Map.Entry<String, String>> originalList);
+
+    /**
+     * Handles the shutdown of the current process if we receive an out of memory error
+     * @param exception out of memory exception
+     * @param tag tag to be included in log statement
+     */
+    void handleShutdownForOutOfMemoryError(@NonNull final BaseException exception, @NonNull final String tag);
```

Comment:
>handleShutdownForOutOfMemoryError [](http://example.com/codeflow?start=9&length=33)

We should only have method likd killProcess(), because that's platform dependent

Reply:
Sorry, not sure what you mean here. are you saying the method should be renamed?

Reply:
Nevermind, I see what you mean